CMakeLists.txt

```diff
@@ -264,7 +264,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Android")
        src/unix/sysinfo-memory.c)
 endif()
 
-if(CMAKE_SYSTEM_NAME MATCHES "Android|Darwin|Linux|OS/390")
+if(CMAKE_SYSTEM_NAME MATCHES "Android|Linux|OS/390" OR APPLE)
```

Comment:
I have a mild preference for writing these as `if(APPLE OR CMAKE_SYSTEM_NAME MATCHES "...")` because otherwise it might be easy to overlook the `OR APPLE` clause.

Reply:
Thanks, I agree with you! I will make the change.